### PR TITLE
Upgrade Django and Elasticsearch

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,15 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+Unreleased
+==========
+
+Changed
+-------
+- **BACKWARD INCOMPATIBLE:** Use Elasticsearch version 5.x
```
Comment (JureZmrzlikar) [inline docs/CHANGELOG.rst]: Isn't it 6.x?

File: docs/composer.rst [hunk 2]
```diff
@@ -82,10 +82,10 @@ defining an extension as follows:
 
     class ExtendedDataDocument:
         """Data ES document extensions."""
-        source = dsl.String(index='not_analyzed')
-        species = dsl.String(index='not_analyzed')
-        build = dsl.String(index='not_analyzed')
-        feature_type = dsl.String(index='not_analyzed')
+        source = dsl.Keyword()
+        species = dsl.Text()
```
Comment (JureZmrzlikar) [inline docs/composer.rst]: Just curious: what is the difference between the `dsl.Text` in `dsl.Keyword`?

File: tests/docker-compose.yml [hunk 3]
```diff
@@ -9,21 +9,11 @@ postgresql:
     ports:
         - "55432:5432"
 elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.8
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.7.0
```
Comment (JureZmrzlikar) [inline tests/docker-compose.yml]: Or is it `6.6.0`?

File: docs/CHANGELOG.rst (hunk 1 above)
Comment (dblenkus) [reply docs/CHANGELOG.rst]: It is :) thanks!

File: docs/composer.rst (hunk 2 above)
Comment (dblenkus) [reply docs/composer.rst]: https://www.elastic.co/blog/strings-are-dead-long-live-strings

File: tests/docker-compose.yml (hunk 3 above)
Comment (dblenkus) [reply tests/docker-compose.yml]: 6.7.0 should be used everywhere, but it doesn't install on travis, so I've used 6.6.0 there until i figure out what is wrong.

**Pull request merged**